lib/explorers/api-parameters.explorer.ts

```diff
@@ -177,12 +177,23 @@ const mapModelsToDefinitons = (parameters, definitions) => {
         }
         const modelName = exploreModelDefinition(param.type, definitions);
         const name = param.name ? param.name : modelName;
+        const schema = {
+            $ref: getDefinitionPath(modelName),
+        };
+        if (param.isArray === true) {
```

Comment:
Use only `if (param.isArray)` like in https://github.com/nestjs/swagger/pull/59/files#diff-c68cb14d77a4b6e5bc3a884b90578cebR152

Reply:
Fixed